shared/src/main/java/com/sap/psr/vulas/shared/json/model/ExemptionBug.java

```diff
@@ -0,0 +1,230 @@
+/**
+ * This file is part of Eclipse Steady.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Copyright (c) 2018 SAP SE or an SAP affiliate company. All rights reserved.
+ */
+package com.sap.psr.vulas.shared.json.model;
+
+import java.util.Arrays;
+import java.util.Iterator;
+import java.util.Map;
+
+import org.apache.commons.configuration.Configuration;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+
+import com.sap.psr.vulas.shared.util.StringUtil;
+
+
+/**
+ * Exemptions are used to prevent that {@link VulnerableDependency}s result in build exceptions during the execution of the report goal.
+ * 
+ * Exemptions can be created for bug identifiers and libraries (by specifying their digests) using the following
+ * format: vulas.report.exempt.&lt;bugId&gt;.&lt;digest&gt; = &lt;reason&gt;
+ * 
+ * The wildcard * can be used to indicate that all bugs for a given digest are exempted (or vice-versa).
+ */
+public class ExemptionBug implements IExemption {
+	
+	private static final Log log = LogFactory.getLog(ExemptionBug.class);
+	
+	private static final String ALL = "*";
+	
+	/** Deprecated configuration prefix. **/
+	public static final String DEPRECATED_CFG_PREFIX = "vulas.report.exceptionExcludeBugs";
+		
+	/** New configuration prefix. **/
+	public static final String CFG_PREFIX = "vulas.report.exemptBug";
```

Comment:
"vulas." should be removed to keep on using configurations without the vulas prefix as it was done until now